prisma/scripts/seed-database/utils.ts

```diff
@@ -307,10 +308,59 @@ export async function uploadImageBucketData(
         );
       }
     }
-  } catch (e) {
+  } catch (e: any) {
     console.log(e);
-    console.error("\nCould not fetch images from pravatar.cc:\n");
-    throw e;
+    console.error(
+      "\nCould not fetch images from pravatar.cc. Continueing with one fallback image.\n"
+    );
+    if (e.cause.code === "ENOTFOUND") {
+      console.error(
+        "Either you have no internet connection or the faker image server is down. Skipped fetching and uploading images to bucket."
+      );
+    }
+    fs.readFile(
```

Comment:
No need to start callback hell. You can use the async variant of `fs` (`node:fs/promises`) or the `fs-extra` package we already use.